test/parallel/test-buffer-alloc.js

```diff
@@ -1009,13 +1010,6 @@ assert.strictEqual(SlowBuffer.prototype.offset, undefined);
   assert.throws(() => Buffer.from(new ArrayBuffer(0), -1 >>> 0), errMsg);
 }
 
-// ParseArrayIndex() should reject values that don't fit in a 32 bits size_t.
-common.expectsError(() => {
-  const a = Buffer.alloc(1);
-  const b = Buffer.alloc(1);
-  a.copy(b, 0, 0x100000000, 0x100000001);
-}, outOfRangeError);
-
```

Comment:
Just because it's not obvious why the test is removed: can you elaborate why it's now obsolete?

Reply:
`ParseArrayIndex()` isn't called without type casting to uint32 any more.

https://github.com/nodejs/node/blob/afeb56adf0c106cc4535d0ee20da078478be7ca2/src/node_buffer.cc#L164